include/BooleanScorer.h

```diff
@@ -87,7 +89,13 @@ class BucketScorer : public Scorer {
     BucketScorer();
     virtual ~BucketScorer();
 
+    
+    int32_t freq;
     LUCENE_CLASS(BucketScorer);
+    float term_freq(){
```

Comment:
Could we rename this to conform to the convention: `termFreq`?

Reply:
763bddeb19711022eb9cb66fe9da452f3ce119c4